### ISSUE TITLE
Cmd Line webapi2swagger: Create missing output directory

### ISSUE THREAD

**Issue by CastleSoft** [NSwag/NSwag#78]
Given the following Post-Build event command line:

```
nswag webapi2swagger /assembly:$(TargetPath) /controller:WebApplication1.Controllers.ValuesController /output:$(ProjectDir)\TypeScript\$(SolutionName).json
nswag swagger2tsclient /input:$(ProjectDir)\TypeScript\$(SolutionName).json /output:$(ProjectDir)\TypeScript\$(SolutionName).ts
```

If the TypeScript folder doesn't exist, nswag fails with an error.

It would be nice if the code created the directory if it didn't exist.

**Comment by rsuter** [comment]
I'm sure there is an msbuild task to create a missing directory... try this until this feature is implemented.

**Comment by jkonecki** [comment]
I believe this issue is fixed and can be closed.

**Comment by rsuter** [comment]
Fixed in v2.33

**Comment by rsuter** [comment]
See https://github.com/NSwag/NSwag/pull/130
